skype/skype-ps/skype/Set-CsTeamsCallingPolicy.md

```diff
@@ -115,6 +115,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -AllowCallRedirect
+Controls call redirection capabilities.
+Turning this on will allow users to redirect calls to target users or voicemail.
```

Comment:
```suggestion
Controls call redirection capabilities.
Turning this on will allow users to redirect calls to target users or voicemail.

Possible values are: Enabled,Disabled,UserOverride.
```

It looks like it's not boolean. And type should be String. Could you check? Thanks

Reply:
Thank you for the review. I validated the cmdlet and it was indeed a String instead of boolean. I updated the documentation accordingly.